lib/elixir/lib/kernel/lexical_tracker.ex

```diff
@@ -101,18 +113,28 @@ defmodule Kernel.LexicalTracker do
     {:reply, Enum.sort(directives), state}
   end
 
-  def handle_call(:remotes, _from, state) do
+  def handle_call(:remote_references, _from, state) do
     {:reply, partition(Enum.to_list(state.references), [], []), state}
   end
 
+  def handle_call(:remote_dispatches, _from, state) do
+    {:reply, partition(Enum.to_list(state.dispatches), [], []), state}
+  end
+
   def handle_call(:dest, _from, state) do
     {:reply, state.dest, state}
   end
 
-  def handle_cast({:remote_dispatch, module, mode}, state) do
+  def handle_cast({:remote_reference, module, mode}, state) do
     {:noreply, %{state | references: add_reference(state.references, module, mode)}}
   end
 
+  def handle_cast({:remote_dispatch, module, fa, line, mode}, state) do
+    references = add_reference(state.references, module, mode)
```

Comment:
maybe this add_reference isn't necessary? I think any references to module names end up in `:remote_references`. Left it here for now, but my testing seems to show that it's unnecessary.